source/manage/health-check.rst

```diff
@@ -0,0 +1,34 @@
+Mattermost Health Check
+========================
+
+.. include:: ../_static/badges/ent-selfhosted.rst
+  :start-after: :nosearch:
+
+The Mattermost Health Check is a comprehensive evaluation of your system’s current operational health. It provides insights based on your system’s configuration, user activity, and log events, helping you maintain optimal performance and reliability.
```

Comment:
Wonder if we want to have more of a "value from Mattermost" to this. Layout directly that we offer this to our Enterprise customers and its importance to Mission Critical Collaboration.